### ISSUE TITLE
Editor is not writable

### ISSUE THREAD

**Issue by xurshid29** [egeloen/IvoryCKEditorBundle#315]
Hi, there is a strange issue with my editor. Editor is not writable If I name my URL like '/banners/new', everything is working fine if I remove the 's' from banners, so everything is working with '/banner/new'.

Symfony 3.3.8
ckeditor-bundle: 6.0.1 (same issue with 5.x.x versions also)

[ckeditor_e.mov.zip](https://github.com/egeloen/IvoryCKEditorBundle/files/1285782/ckeditor_e.mov.zip)

**Comment by egeloen** [comment]
Weird... Any error in the browser console? Have you tried to clear your browser cache?

**Comment by xurshid29** [comment]
Yes, it's so weird. No errors in the console. Tried to clear the caches, also tried with incognito mode, tried with another browser, no luck.

**Comment by egeloen** [comment]
The issue is I'm not able to reproduce it locally, so it's hard for me to help you...

**Comment by validaide-marcel-tuinstra** [comment]
Same issue but when click on one of the buttons the following issue apears in console.
TypeError: this.document.getWindow(...).$

Might be related to the textarea not writeable?
